packages/replay/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "bootstrap": "yarn && cd demo && yarn #TODO: change after migration",
     "build": "yarn build:rollup",
```

Comment:
hmm, will this work? As `build` calls `build:rollup`? 😅 I guess either change `build` to call `build:extra`, or remove it? (not sure)

Reply:
good point. I think for CI we'd be fine because `build` isn't called from the CI (the top-level `yarn build` executes `build:rollup` in every package)